conditional_camera/conditional_cam.go

```diff
@@ -0,0 +1,253 @@
+package conditional_camera
+
+import (
+	"context"
+	"fmt"
+	"image"
+	"sort"
+	"sync"
+	"time"
+
+	"go.viam.com/rdk/components/camera"
+	"go.viam.com/rdk/data"
+	"go.viam.com/rdk/logging"
+	"go.viam.com/rdk/pointcloud"
+	"go.viam.com/rdk/resource"
+	"go.viam.com/rdk/services/generic"
+
+	"go.viam.com/rdk/gostream"
+	"go.viam.com/utils"
+)
+
+var Model = resource.ModelNamespace("viam").WithFamily("camera").WithModel("conditional-camera")
+
+type Config struct {
+	Camera        string `json:"camera"`
+	FilterSvc     string `json:"filter_svc"`
```

Comment:
can we spell out "filter_service" fully for the json? Makes it more clear to a user using this for the first time.